templates/taxi/driver_list.html

```diff
@@ -1,34 +1,37 @@
-{% extends "base.html" %}
-
-{% block content %}
-    <h1>
-      Driver List
-      <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
-        Create
-      </a>
-    </h1>
-
-    {% if driver_list %}
-    <table class="table">
-      <tr>
-        <th>ID</th>
-        <th>Username</th>
-        <th>First name</th>
-        <th>Last name</th>
-        <th>License number</th>
-      </tr>
-    {% for driver in driver_list %}
-      <tr>
-        <td>{{ driver.id }}</td>
-        <td><a href="{{ driver.get_absolute_url }}">{{ driver.username }} {% if user == driver %} (Me){% endif %}</a></td>
-        <td>{{ driver.first_name }}</td>
-        <td>{{ driver.last_name }}</td>
-        <td>{{ driver.license_number }}</td>
-      </tr>
-    {% endfor %}
-
-    </table>
-    {% else %}
-      <p>There are no drivers in the service.</p>
-    {% endif %}
-{% endblock %}
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>
+      Driver List
+      <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
+        Create
+      </a>
+    </h1>
+    <form method="get" action="{% url 'taxi:search-drivers' %}">
+      <input type="text", name="q", placeholder="Search drivers (username)">

```

Comment:
There is a comma after the `name` attribute in the input tag. This should be removed to ensure proper HTML syntax.